### PR TITLE
:warning:  Switch OSP default to 17.1 :warning:

### PR DESCRIPTION
**Pull request by EmilienM**
* Update README and add RHEL versions table
* Move telco examples to 17.1
* Add new example for 16.2
* Updade vars defaults and add some doc too for storage on RHEL9
* Skip some tasks in prepare_host for RHEL9
* Add `ntp_server` parameter (with a safe default when not using RHSM) for deploying with a specific NTP.
* Handle the RH container registry login when deploying Ceph.
  Unfortunately, `redhat_registry_credentials` is now a dict with mandatory `username` and `password` fields.
* Make cephadm config working for DCN.
* Allow to expose `ceph_devices` via LVM.

### CODE REVIEW

**Comment by EmilienM** [comment]
Some of us are currently testing it, I suggest that we wait for it to be merged.

Note that this brings backward incompatible changes to dev-install; so I'll produce a release today which will be usable for anyone who wants the default to work with 16.2 and RHEL 8.4.

**Comment by EmilienM** [comment]
I need to add support for NtpServer when using internal puddles. That's for Ceph; I'll explain in a next patchset, tomorrow.

File: example-overrides/local-overrides-osp16-2-with-dpdk.yaml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-dpdk_interface: enp130s0f0
```
Comment (MaysaMacedo) [inline example-overrides/local-overrides-osp16-2-with-dpdk.yaml:1]: why is this example being removed?

File: playbooks/vars/defaults.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-rhos_release: 16.2
+rhos_release: 17.1
```
[the remark this answers is not in the document]
Comment (EmilienM) [reply playbooks/vars/defaults.yaml:2]: yes I know, but at the same time I don't think we should wait for the next FDP release (October AFIK).
dev-install users who want FDP should know that it's currently not working now in 17.1...

File: example-overrides/local-overrides-osp16-2-with-dpdk.yaml (hunk 1 above)
Comment (EmilienM) [reply example-overrides/local-overrides-osp16-2-with-dpdk.yaml:1]: it's renamed for 17.1. No point of duplicating examples for DPDK.

**Comment by MaysaMacedo** [comment]
@EmilienM Is it ready for review?

**Comment by EmilienM** [comment]
> @EmilienM Is it ready for review?

Yes, I'm debugging something today but I'm not sure it involves dev-install.

**Comment by EmilienM** [comment]
This is ready for review, @gprocunier had some comments (offline) but I think this can be addressed on a separate PR.

**Comment by EmilienM** [comment]
further fixes/improvements should come via another PR.

Thanks for the review, let's iterate.

**Pull request merged**